internal/manifests/collector/ports.go

```diff
@@ -21,50 +21,36 @@ import (
 )
 
 const (
-	StatsD             = "statsd"
-	CollectD           = "collectd"
-	XrayProxy          = "aws-proxy"
-	XrayTraces         = "aws-traces"
-	OtlpGrpc           = "otlp-grpc"
-	OtlpHttp           = "otlp-http"
-	AppSignalsGrpc     = "appsig-grpc"
-	AppSignalsHttp     = "appsig-http"
-	AppSignalsProxy    = "appsig-xray"
-	AppSignalsGrpcSA   = ":4315"
-	AppSignalsHttpSA   = ":4316"
-	AppSignalsProxySA  = ":2000"
-	AppSignalsServerSA = ":4311"
-	EMF                = "emf"
-	EMFTcp             = "emf-tcp"
-	EMFUdp             = "emf-udp"
-	CWA                = "cwa-"
-	JmxHttp            = "jmx-http"
-	Server             = "server"
+	StatsD          = "statsd"
+	CollectD        = "collectd"
+	XrayProxy       = "aws-proxy"
+	XrayTraces      = "aws-traces"
+	OtlpGrpc        = "otlp-grpc"
+	OtlpHttp        = "otlp-http"
+	AppSignalsGrpc  = CWA + "appsig-grpc"
+	AppSignalsHttp  = CWA + "appsig-http"
+	AppSignalsProxy = CWA + "appsig-xray"
+	EMF             = "emf"
+	EMFTcp          = "emf-tcp"
+	EMFUdp          = "emf-udp"
+	CWA             = "cwa-"
+	JmxHttp         = "jmx-http"
+	Server          = CWA + "server"
```

Comment:
Adding `CWA +` to retain previous naming, but if it's okay to change these names, then I can rename them without adding `CWA +`.